packages/vuetify/src/components/VChip/VChip.tsx

```diff
@@ -98,22 +100,31 @@ export const VChip = defineComponent({
     const isActive = useProxiedModel(props, 'modelValue')
     const group = useGroupItem(props, VChipGroupSymbol, false)
     const link = useLink(props, attrs)
+    const isClickable = computed(() => !props.disabled && (!!group || link.isClickable.value || props.link))
 
     function onCloseClick (e: Event) {
       isActive.value = false
 
       emit('click:close', e)
     }
 
+    function onClick (e: MouseEvent) {
+      emit('click', e)
+
+      if (!isClickable.value) return
+
+      link.navigate?.(e)
+      group?.toggle()
+    }
+
     return () => {
       const Tag = (link.isLink.value) ? 'a' : props.tag
       const hasAppend = !!(slots.append || props.appendIcon || props.appendAvatar)
       const hasClose = !!(slots.close || props.closable)
       const hasFilter = !!(slots.filter || props.filter) && group
       const hasPrepend = !!(slots.prepend || props.prependIcon || props.prependAvatar)
       const hasColor = !group || group.isSelected.value
-      const isClickable = !props.disabled && (!!group || link.isClickable.value || props.link)
-      const onClickFunc = props.link ? props.link : group?.toggle
+      const onClickFunc = onClick
```

Comment:
Doesn't need to be aliased

Reply:
As in just pass `onClick` into the component?

Reply:
Yeah just use onClick directly